### PR TITLE
Lr-parallel-64 - fix building on rpi4/kms

### PR DESCRIPTION
**Pull request by joolswills**
* remove a typedef conflict - may need to upstream a proper fix for this
 * change logic from "tinker" to "kms" for make parameters to make it more general
 * rollback code for "neon" rather than "arm" platforms

I've not tested this at all apart from building on rpi2 and rpi4. Need to check the typedef change on tinkerboard. Probably should be upstreamed but upstream master is broken on arm which also needs fixing.

### CODE REVIEW

**Comment by joolswills** [comment]
To clarify. HEAD doesn't build on arm due to some makefile changes or so. Fails at linking stage.

https://github.com/libretro/parallel-n64/issues/544

Probably a simple fix.

File: scriptmodules/libretrocores/lr-parallel-n64.sh [hunk 1]
```diff
@@ -28,18 +28,20 @@ function sources_lr-parallel-n64() {
...
 }
 
 function build_lr-parallel-n64() {
     rpSwap on 1000
     make clean
     local params=()
-    if isPlatform "rpi" || isPlatform "odroid-c1"; then
-        params+=(platform="$__platform")
-    elif isPlatform "tinker"; then
+    if isPlatform "kms"; then
```
Comment (psyke83) [inline scriptmodules/libretrocores/lr-parallel-n64.sh]: Can you change this to `mesa`, please?

IMHO, usage of the `kms` platform flag should be architecture-agnostic when possible due to the fact that any modern system is potentially capable of running in KMS mode. Your PR runs fine on x86 kms mode with the flag changed to `mesa` (whereas the PR breaks the x86 kms build as-is). Thanks.

Comment (joolswills) [reply scriptmodules/libretrocores/lr-parallel-n64.sh]: Thanks. Will do.

**Comment by joolswills** [comment]
Merging but this may need further platform logic tweaks.

**Pull request merged**